pipeline/transformer/transformer_common/src/main/scala/weco/pipeline/transformer/identifiers/SourceIdentifierValidation.scala

```diff
@@ -0,0 +1,65 @@
+package weco.pipeline.transformer.identifiers
+
+import grizzled.slf4j.Logging
+import weco.catalogue.internal_model.identifiers.{
+  IdentifierType,
+  SourceIdentifier
+}
+
+import java.util.UUID
+import scala.Function.const
+import scala.util.Try
+import scala.util.matching.Regex
+
+object SourceIdentifierValidation {
+  implicit class SourceIdentifierOps(sourceIdentifier: SourceIdentifier)
+      extends Logging {
+    import IdentifierRegexes._
+
+    def validated: Option[SourceIdentifier] =
```

Comment:
Do we ever use this outside of tests?

Reply:
Yes - https://github.com/wellcomecollection/catalogue-pipeline/blob/14674e27fd87910a1eb40658b3e937cd9a79c708/pipeline/transformer/transformer_miro/src/main/scala/weco/pipeline/transformer/miro/transformers/MiroIdentifiers.scala#L89